book/src/proptest-derive/modifiers.md

```diff
@@ -0,0 +1,370 @@
+# Modifier Reference
+
+All modifiers interpreted by `#[derive(Arbitrary)]` are of the form
+`#[proptest(..)]`, where the content between the parentheses follows the normal
+Rust attribute syntax.
+
+Each modifier within the parentheses is independent, in that putting two
+modifiers in the same attribute is equivalent to having two `#[proptest(..)]`
+attributes with one modifier each.
+
+For brevity, modifiers are sometimes referenced by name alone; e.g., "the
+`weight` modifier" refers to `#[proptest(weight = nn)]` and not some
+freestanding `#[weight]` attribute.
+
+## `filter`
+
+Form: `#[proptest(filter = F)]` or `#[proptest(filter(F))]` where `F` is either
+a bare identifier or a Rust expression in a string. In either case, the
```

Comment:
Clarify that "bare identifier" refers to a function (or const/static also works...)?